sdk/identity/identity/.eslintrc.json

```
@@ -1,7 +1,7 @@
 {
   "plugins": ["@azure/azure-sdk"],
   "extends": ["plugin:@azure/azure-sdk/azure-sdk-base"],
-  "ignorePatterns": ["src/core"],
+  "ignorePatterns": ["src/core", "test/manual*"],
```

why are you ignoring those directories? are they autogenerated?

---

also where does the autogenerated files live? I do not see a `src/core` anywhere, so perhaps you wanted to delete that pattern.
```suggestion
  "ignorePatterns": ["test/manual*"],
```